drivers/led/led_dac.h

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2025 Prevas A/S
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_DRIVERS_LED_DAC_H_
+#define ZEPHYR_INCLUDE_DRIVERS_LED_DAC_H_
+
+#include <zephyr/device.h>
+
+int led_dac_set_raw(const struct device *dev, uint32_t led, uint32_t value);
```

Comment:
The header is not required, the function is only used internally anyway :) if this was to be public it would need to go to `include/zephyr/drivers/led`, but I don't think this is necessary :)

Reply:
You are right. It an attempt as a work around until https://github.com/zephyrproject-rtos/zephyr/issues/87737 is resolved/gets response.

Reply:
Header removed and function made `static`.